### ISSUE TITLE
[Visual Studio Code] Configuring Pipenv in Visual Studio Code （在VS Code中配置Pipenv）

### ISSUE THREAD

**Issue by hengzhou23** [hengzhou23/tutorials#1]
Pipenv is a more modern way to manage project dependencies in Python. However, if you want to use Visual Studio Code's Python plugin for your project, you need to tell it where it can find your virtualenv.

First, navigate to your Pipenv directory (where your Pipfile is located). Then run `pipenv --venv`, to get the full path to the Pipenv's virtualenv. This is probably something like `/home/myuser/.local/share/virtualenvs/projectname` (or maybe `/user/local/bin/python3`), depending on which operating system you are on.

Next, press **CTRL+Shift+P** in VS Code (**Cmd+Shift+P** if you are on MacOS) and search for "Select Workspace Interpreter" (or try searching "Python: Select Interpreter"). Press enter, and select any of the suggested interpreters (just select a random one as you will change it later in this tutorial). Upon doing this, a `.vscode` directory will be created inside your **project root**, containing a `settings.json`. Open this file, and set `python.pythonPath` to the virtualenv path you found earlier, and add `/bin/python` at the end.

In other words, settings.json should look something like this:

```
{
    "python.pythonPath": "/home/myuser/.local/share/virtualenvs/projectname/bin/python"
}
```

Save the settings file, and restart Visual Studio Code, for the changes to take effect.

Original article: <https://olav.it/2017/03/04/pipenv-visual-studio-code/>
Original author: [Olav Lindekleiv] (https://github.com/oal)

Edited by Harry Zhou (added comments in brackets)

**Comment by hengzhou23** [comment]
Also you can use `pipenv --py` to get the absolute path of python interpreter in current pipenv environment. This might be easier :)
